src/eduid/userdb/group_management/db.py

```diff
@@ -94,51 +92,29 @@ def get_states_by_email_addresses(self, email_addresses: List[str]) -> List[Grou
 
         :raise self.DocumentDoesNotExist: No document match the search criteria
         """
-        states = []
+        states: list[GroupInviteState] = []
         for email_address in email_addresses:
             spec = {"email_address": email_address}
-            states.extend(list(self._get_documents_by_filter(spec)))
+            for this in self._get_documents_by_filter(spec):
```

Comment:
"this" kanske är en python grejj, om det inte är så kanske "user_document" är ett lämpligt namn?